README.md

```diff
@@ -46,6 +49,15 @@ async fn handle_streamer_message(
 }
 ```
 
+Video tutorial:
+
+https://youtu.be/GsF7I93K-EQ
+
+### More examples
+
+- https://github.com/near/near-lake-raw-printer simple example of a data printer built on top of NEAR Lake Framework
```

Comment:
Do you have permissions to move printer to near-examples org? I think it makes sense to keep it there for now

Reply:
did it